src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -204,22 +204,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     packages references.
     ============================================================
     -->
-  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectResolvedSDKReferencesDesignTime"
-             AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  
-  <Target Name="CollectSDKReferencesDesignTime"
-          Returns="@(_SDKReference)"
-          DependsOnTargets="ResolvePackageDependenciesDesignTime">
+  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectSDKReferencesDesignTime"
+         AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
-    <CollectResolvedSDKReferencesDesignTime
-          ResolvedSdkReferences="@(SdkReference)"
-          DependenciesDesignTime="@(_DependenciesDesignTime)">
+  <Target Name="CollectSDKReferencesDesignTime"
+          Returns="@(_SDKReference)">
```

Comment:
Should this be run after something like `CollectPackageReferences` so that targets can add or remove package references before they are processed here?